<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestion_reclamation</class>
 <widget class="QDialog" name="Gestion_reclamation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#Gestion_reclamation{
background-image: url(:/new/background/background);
}

#Reclamations{
background-color:rgba(133, 207, 194,0.5);
}

QLineEdit{
border:none;
border-bottom:1px solid red;
}</string>
  </property>
  <widget class="QTabWidget" name="STAT">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>981</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Reclamations">
    <attribute name="title">
     <string>Reclamation</string>
    </attribute>
    <widget class="QToolBox" name="ensemble_abonnees">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>821</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
</string>
     </property>
     <widget class="QWidget" name="ensemble_abonneesPage1_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Ajouter</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_ajout">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>641</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
</string>
       </property>
       <property name="title">
        <string>Ajout</string>
       </property>
       <widget class="QLabel" name="label_ID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID reclamation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>260</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_date">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date reclamation</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="place">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="lineEdit_date">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cas</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cas">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>160</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="mail_verif">
        <property name="geometry">
         <rect>
          <x>356</x>
          <y>210</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage2_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_reclamation">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>741</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>21</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>22</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
       </property>
       <item>
        <property name="text">
         <string>IDRECLAMATION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EMAIL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATERECLAMATION</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>22</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Desc</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage3_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Supprimer</string>
      </attribute>
      <widget class="QGroupBox" name="ensemble_aff">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>601</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>Supression</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_IDSUP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>102</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_IDSUP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID reclamation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>101</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage4_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Modifier</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Modification</string>
       </property>
       <widget class="QLineEdit" name="lineEdit_idmodif">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>101</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_IDmod">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID reclamation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_casmodif">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_IDmod_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Cas</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>100</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Reservation</string>
    </attribute>
    <widget class="QToolBox" name="ensemble_abonnees_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>821</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
</string>
     </property>
     <widget class="QWidget" name="ensemble_abonneesPage1_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Ajouter</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_ajout_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>641</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Ajout</string>
       </property>
       <widget class="QLabel" name="label_ID_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>290</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_date_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ID">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="place_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="adresse">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>160</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="mail_verif_3">
        <property name="geometry">
         <rect>
          <x>356</x>
          <y>210</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="Produit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="place_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRODUIT</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>115</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage2_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_reservation">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>741</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>21</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>22</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
       </property>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ADRESSE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PRODUIT</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>22</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Desc</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage3_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Supprimer</string>
      </attribute>
      <widget class="QGroupBox" name="ensemble_aff_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>601</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>Supression</string>
       </property>
       <widget class="QLineEdit" name="ID_SUP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>102</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_IDSUP_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_8">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>101</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="ensemble_abonneesPage4_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>363</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Modifier</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Modification</string>
       </property>
       <widget class="QLabel" name="label_IDmod_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID reclamation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_9">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#db0003;
border-radius:10px;
color:white;</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_date_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_2">
       <property name="geometry">
        <rect>
         <x>233</x>
         <y>111</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_2">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_ID_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="place_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRODUIT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>181</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="adresse_2">
       <property name="geometry">
        <rect>
         <x>231</x>
         <y>174</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Produit_2">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
